addons/website_sale/views/templates.xml

```diff
@@ -986,19 +1020,95 @@
+        </xpath>
+        <xpath expr="//div[@t-if='a.value_ids and len(a.value_ids) &gt; 1']" position="attributes">
+            <attribute name="t-attf-class">accordion-item rounded-0</attribute>
+        </xpath>
+        <xpath expr="//div[@t-if='a.value_ids and len(a.value_ids) &gt; 1']//h6" position="replace">
+            <h6 class="accordion-header">
+                <button
+                    class="accordion-button px-0 bg-transparent shadow-none"
+                    type="button"
+                    data-bs-toggle="collapse"
+                    t-attf-data-bs-target="#o_products_attributes_{{a.id}}"
+                    t-attf-aria-controls="o_products_attributes_{{a.id}}" aria-expanded="true"
+                >
+                    <b class="o_products_attributes_title" t-field="a.name"/>
+                </button>
+            </h6>
+        </xpath>
+        <xpath expr="//div[@t-attf-id='o_products_attributes_{{a.id}}']" position="attributes">
+            <attribute name="class" add="accordion-collapse collapse show" separator=" "/>
+            <attribute name="data-bs-parent" add="wsale_products_attributes_collapse"/>
+        </xpath>
+        <xpath expr="//select" position="attributes">
+            <attribute name="class" add="mb-3" remove="mb-2" separator=" "/>
+        </xpath>
+        <xpath expr="//div[hasclass('flex-column')]" position="attributes">
+            <attribute name="class" add="mb-3" separator=" "/>
+        </xpath>
+    </template>
+
+    <template id="filter_products_price" name="Filter by Prices">
         <t t-set="isDisabled" t-value="available_min_price == available_max_price"/>
-        <div id="o_wsale_price_range_option"
-             t-attf-class="position-relative {{_classes}} {{isDisabled and 'opacity-75 pe-none user-select-none'}}">
-            <label t-attf-class="m-0 h6 o_products_attributes_title {{_classes_title}}">
-                <b>Price Range</b>
-            </label>
-            <input type="range" multiple="multiple"
-                   t-attf-class="form-range range-with-input {{_classes_input}}"
-                   t-att-data-currency="website.currency_id.symbol"
-                   t-att-data-currency-position="website.currency_id.position"
-                   t-att-step="website.currency_id.rounding" t-att-min="'%f' % (available_min_price)"
-                   t-att-max="'%f' % (available_max_price)" t-att-value="'%f,%f' % (min_price, max_price)"/>
+        <t
+            t-set="isCollapsed"
+            t-value="is_view_active('website_sale.products_attributes_collapsed') and canBeCollapsed"
+        />
+        <div
+            id="o_wsale_price_range_option"
+            t-attf-class="position-relative {{_classes}} {{isDisabled and 'opacity-75 pe-none user-select-none'}} {{isCollapsed and 'accordion accordion-flush border-top'}}"
+        >
+            <t t-if="isCollapsed">
```

Comment:
`is_collapsible`? I know that the option-enabled views have names like something_collapsed, but I think these views, and this current viable, should use "_collapsible".